runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -133,160 +138,41 @@ public void mergeSegments(@Nonnull String endpointToRestoreRedundancy, int retry
-     *   cluster status report as clusterStatusReliability.
-     *
+     * layout. This information is presented along each node's status (up, down, db_sync).
+     * <p>
+     * It is important to note that as the cluster state is obtained from the layout,
+     * when quorum is not available (majority of nodes) there are lower guarantees on the
+     * reliability of this state.
+     * For example, in the absence of quorum the system might be in an unstable state which
+     * cannot converge due to lack of consensus. This is reflected in the
+     * cluster status report as clusterStatusReliability.
+     * <p>
      * - (2) The connectivity status of the client to every node in the cluster,
-     *   i.e., can the client connect to the cluster. This will be obtained by
-     *   ping(ing) every node and show as RESPONSIVE, for successful connections or UNRESPONSIVE for
-     *   clients unable to communicate.
-     *
-     *  In this sense a cluster can be STABLE with all nodes UP, while not being available for a
-     *  client, as all connections from the client to the cluster nodes are down, showing in this
-     *  case connectivity status to all nodes as UNRESPONSIVE.
-     *
-     *  The ClusterStatusReport consists of the following:
-     *
-     *  CLUSTER-SPECIFIC STATUS
-     *  - clusterStatus: the cluster status a perceived by the system's layout.
-     *  STABLE, DEGRADED, DB_SYNCING or UNAVAILABLE
-     *  - nodeStatusMap: each node's status as perceived by the system's layout.
-     *  (UP, DOWN or DB_SYNC)
-     *  - Cluster Status Reliability: STRONG_QUORUM, WEAK_NO_QUORUM or UNAVAILABLE
-     *
-     *  CLIENT-CLUSTER SPECIFIC STATUS:
-     *  - clientServerConnectivityStatusMap: the connectivity status of this client to the cluster.
-     *    (RESPONSIVE, UNRESPONSIVE).
+     * i.e., can the client connect to the cluster. This will be obtained by
+     * ping(ing) every node and show as RESPONSIVE, for successful connections or UNRESPONSIVE for
+     * clients unable to communicate.
+     * <p>
+     * In this sense a cluster can be STABLE with all nodes UP, while not being available for a
+     * client, as all connections from the client to the cluster nodes are down, showing in this
+     * case connectivity status to all nodes as UNRESPONSIVE.
+     * <p>
+     * The ClusterStatusReport consists of the following:
+     * <p>
+     * CLUSTER-SPECIFIC STATUS
+     * - clusterStatus: the cluster status a perceived by the system's layout.
+     * STABLE, DEGRADED, DB_SYNCING or UNAVAILABLE
+     * - nodeStatusMap: each node's status as perceived by the system's layout.
+     * (UP, DOWN or DB_SYNC)
+     * - Cluster Status Reliability: STRONG_QUORUM, WEAK_NO_QUORUM or UNAVAILABLE
+     * <p>
+     * CLIENT-CLUSTER SPECIFIC STATUS:
+     * - clientServerConnectivityStatusMap: the connectivity status of this client to the cluster.
+     * (RESPONSIVE, UNRESPONSIVE).
```

Comment:
Can you add an example output in the comment?  It is not clear what all info is displayed/collected..

Reply:
Didn't get your question. There could be only one of those two states RESPONSIVE or UNRESPONSIVE